plugins/modules/datadog_monitor.py

```diff
@@ -167,6 +167,32 @@
           - Integer from 1 (high) to 5 (low) indicating alert severity.
         type: int
         version_added: 4.6.0
+    notification_preset_name:
+        description:
+          - Toggles the display of additional content sent in the monitor notification.
+        choices:
+            - show_all
+            - hide_query
+            - hide_handles
+            - hide_all
+        type: str
+        version_added: 7.1.0
+    renotify_occurrences:
+        description:
+            - The number of times re-notification messages should be sent on the current status at the provided re-notification interval.
+        type: int
+        version_added: 7.1.0
+    renotify_statuses:
+        description:
+            - The types of monitor statuses for which re-notification messages are sent. Valid values are alert, warn, no data.
```

Comment:
Since they are already listed under `choices`, there's no need to list them here again.
```suggestion
            - The types of monitor statuses for which re-notification messages are sent.
```